### ISSUE TITLE
Anope 2.1.4 crashes on startup

### ISSUE THREAD

**Issue by Madriix** [anope/anope#391]
Here is the log ./anope --support : 

```
> [Apr 04 19:36:01.450218 2024] Debug: Received @realname=/F/59;unrealircd.org/geoip=FR;unrealircd.org/userhost=motus@ns.discussionner.com;unrealircd.org/userip=motus@213.186.61.115;account=Marie;msgid=dLnIT09dahGn9EdP27Cmhn;time=2024-04-04T17:36:01.449Z :00131913E MODE #motus -ee *!bot02*@* *!*@*.discussionner.com
> [Apr 04 19:36:01.450420 2024] Debug: Sent :00A ERROR :Invalid timestamp: *!*@*.discussionner.com
> [Apr 04 19:36:02.093315 2024] Debug: Protocol error: Invalid timestamp: *!*@*.discussionner.com
> [Apr 04 19:36:02.093348 2024] Debug: Sent :00AAAAAAD PRIVMSG #!opers! :Protocol error: Invalid timestamp: *!*@*.discussionner.com
> [Apr 04 19:36:02.093890 2024] Debug: Sent :00AAAAAAD QUIT :Shutting down
> [Apr 04 19:36:02.093918 2024] Debug: Sent :00AAAAAAI QUIT :Shutting down
> [Apr 04 19:36:02.093935 2024] Debug: Sent :00AAAAAAM QUIT :Shutting down
> [Apr 04 19:36:02.093953 2024] Debug: Sent :00AAAAAAQ QUIT :Shutting down
> [Apr 04 19:36:02.093971 2024] Debug: Sent :00AAAAAAO QUIT :Shutting down
> [Apr 04 19:36:02.093988 2024] Debug: Sent :00AAAAAAE QUIT :Shutting down
> [Apr 04 19:36:02.094006 2024] Debug: Sent :00AAAAAAB QUIT :Shutting down
> [Apr 04 19:36:02.094023 2024] Debug: Sent :00AAAAAAC QUIT :Shutting down
> [Apr 04 19:36:02.094041 2024] Debug: Sent :00AAAAAAF QUIT :Shutting down
> [Apr 04 19:36:02.094060 2024] Debug: Sent :00AAAAAAH QUIT :Shutting down
> [Apr 04 19:36:02.094079 2024] Debug: Sent :00AAAAAAL QUIT :Shutting down
> [Apr 04 19:36:02.094095 2024] Debug: Sent :00AAAAAAG QUIT :Shutting down
> [Apr 04 19:36:02.094117 2024] Debug: Sent :00AAAAAAR QUIT :Shutting down
> [Apr 04 19:36:02.094135 2024] Debug: Sent :00AAAAAAJ QUIT :Shutting down
> [Apr 04 19:36:02.094159 2024] Debug: Sent :00AAAAAAN QUIT :Shutting down
> [Apr 04 19:36:02.094176 2024] Debug: Sent :00AAAAAAK QUIT :Shutting down
> [Apr 04 19:36:02.094193 2024] Debug: Sent :00AAAAAAP QUIT :Shutting down
> [Apr 04 19:36:02.094213 2024] Debug: Sent :00A SQUIT 00A :Protocol error: Invalid timestamp: *!*@*.discussionner.com
> [Apr 04 19:36:02.094695 2024] SERVER: irc.discussionner.com (Tchat Discussionner) quit from services.discussionner.com for services.discussionner.com irc.discussionner.com
> [Apr 04 19:36:02.094714 2024] Debug: Attempted to send "PRIVMSG" from OperServ when not introduced
> [Apr 04 19:36:02.094735 2024] Debug: Attempted to send "PRIVMSG" from OperServ when not introduced
> [Apr 04 19:36:02.095605 2024] Debug: Finished removing all users for irc.discussionner.com
> [Apr 04 19:36:02.095626 2024] SERVER: services.discussionner.com (Services for IRC Networks) quit irc.discussionner.com
> [Apr 04 19:36:02.095640 2024] Debug: Attempted to send "PRIVMSG" from OperServ when not introduced
> [Apr 04 19:36:02.095662 2024] Debug: Attempted to send "PRIVMSG" from OperServ when not introduced
> [Apr 04 19:36:02.102269 2024] Debug: SQL successfully executed query: INSERT INTO `anope_db_BotInfo` (`id`,`created`
> 
> ...
> 
> [Apr 04 19:36:02.168235 2024] Debug: Unloading module help
> [Apr 04 19:36:02.168286 2024] Debug: Unloading module helpchan
> [Apr 04 19:36:02.168330 2024] Debug: Unloading module sasl
> [Apr 04 19:36:02.168409 2024] Debug: Unloading module cs_fantasy_stats
> [Apr 04 19:36:02.168471 2024] Debug: Unloading module cs_fantasy_top
> [Apr 04 19:36:02.168570 2024] Debug: Unloading module httpd
> [Apr 04 19:36:02.168652 2024] Debug: Unloading module mysql
> [Apr 04 19:36:02.205849 2024] Debug: Error executing query SQL Interface is going away:
> [Apr 04 19:36:02.205908 2024] Debug: Error executing query SQL Interface is going away:
> Erreur de segmentation
> irc@ns:~/anope2/services/bin$
```

**Comment by SadieCat** [comment]
This should be fixed on the latest 2.1 git. Can you check?

**Comment by Madriix** [comment]
@SadieCat Looks like it worked well, thank you. I came to IRC Anope the day before yesterday to talk to you about that, my nickname was "Mi92"
